runtime/cfdumper/main.c

```diff
@@ -907,6 +907,16 @@ static void dumpAttribute(J9CfrClassFile* classfile, J9CfrAttribute* attrib, U_3
 			}
 			break;
 
+		case CFR_ATTRIBUTE_PermittedSubclasses:
+			for(i = 0; i < ((J9CfrAttributePermittedSubclasses*)attrib)->numberOfClasses; i++) {
+				U_16 classIndex = ((J9CfrAttributePermittedSubclasses*)attrib)->classes[i];
+				U_16 nameIndex = classfile->constantPool[classIndex].slot1;
+
+				for(j = 0; j < tabLevel + 1; j++) j9tty_printf( PORTLIB, "  ");
+				j9tty_printf( PORTLIB, "PermittedSubclass class index, name: %i, %i -> %s\n", classIndex, nameIndex, classfile->constantPool[nameIndex].bytes);
```

Comment:
Is the string guaranteed to be nul-terminated? I see this pattern repeated throughout the code, so maybe it is.